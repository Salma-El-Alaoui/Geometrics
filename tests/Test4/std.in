PL pl1 1 1
PL pl2 2 2 2 2
PL pl3 3 3 3 3 3 3
PL pl2 1 2 3 4
PL pl4 a 1 1 1
PL pl4 -1 -1 -1 -1
PL pl5 1 1 1
PL pl5 1 1 1 1 1 1 1
PL 123 1 2 3 4 5 6 7 8
EXIT
