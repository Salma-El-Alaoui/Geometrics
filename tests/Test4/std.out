OK
#Element ajoute: pl1
OK
#Element ajoute: pl2
OK
#Element ajoute: pl3
ERR
#Un element geometrique nomme pl2 existe deja
ERR
#Parametres invalides
OK
#Element ajoute: pl4
ERR
#Parametres invalides
ERR
#Parametres invalides
ERR
#Parametres invalides
